.github/workflows/pull_request_check_revert_candidate_commits.yaml

```diff
@@ -0,0 +1,51 @@
+name: PR Check revert candidate commits
```

Comment:
suggestion: '[Pull Request] Check Revert Candidate Commits' might be better for convention!

Reply:
I agree Thanks @WANZARGEN